### ISSUE TITLE
Workflow Tutorial: Updating Your Databases

### ISSUE THREAD

**Issue by carolhunt** [Datical/liquibase-docs#64]
We need a general workflow tutorial with step-by-step instructions on how to update your database. 

Below is a link to our current documentation that needs to be re-written with step-by-step instructions:
https://docsstage.liquibase.com/workflows/liquibase-community/updating-the-database.html

Below a link to our workflow template:
https://docs.google.com/document/d/1Waz3_OPfQtSQ-WzGbP-hbQW9fFSxQgwS/edit

Please utilize the template to aid in the flow of your work.

Your byline, photo and link to your preferred social page will be included once completed.

**Comment by varadpoddar** [comment]
Hi, I'd like to work on this. Could you assign this issue to me, please?

**Comment by carolhunt** [comment]
It's yours! I just changed the label to hacktoberfest-claimed and assigned this issue to you. Let me know if you have any questions! Thanks for contributing.

**Comment by varadpoddar** [comment]
Thank you !
Am I to reword the tutorial linked [here](https://docsstage.liquibase.com/workflows/liquibase-community/updating-the-database.html) in a step by step manner?

Or rewrite the html with the actual process of updating the database?

If it's the second, could you explain the process so I can write it accordingly? 

😄

**Comment by carolhunt** [comment]
Yes! Re-wording the tutorial but in a more step-by-step manner. 
It should also include running the SQL command for verification because we want our users getting in the habit of verifying their changes before running them :) 
We want to tell the user to configure their properties files, then update their changelog with the changes they want, then verify those changes by running updateSQL, then actually run the update command.

Let me know if you have any other questions!

**Comment by carolhunt** [comment]
Hello @varadpoddar I wanted to check to see if you still had interest in helping with this documentation or if you had any additional questions? Thanks!
